src/context/WalletProvider/KeepKeyProvider.tsx

```diff
@@ -0,0 +1,157 @@
+import { isKeepKey, KeepKeyHDWallet } from '@shapeshiftoss/hdwallet-keepkey'
+import React, {
+  createContext,
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useReducer
+} from 'react'
+import { RadioOption } from 'components/Radio/Radio'
+import { useWallet } from 'hooks/useWallet/useWallet'
+
+export enum DeviceTimeout {
+  TenMinutes = '600000',
+  FifteenMinutes = '900000',
+  TwentyMinutes = '1200000',
+  ThirtyMinutes = '1800000',
+  FortyFiveMinutes = '2700000',
+  SixtyMinutes = '3600000'
+}
+
+export const timeoutOptions: RadioOption<DeviceTimeout>[] = [
```

Comment:
```suggestion
export const timeoutOptions: RadioOption<DeviceTimeout>[] = Object.freeze([
```

Reply:
Addressed in https://github.com/shapeshift/web/pull/1341/commits/543178a225b970f55f09b7736d44c9209f70c03d